inginious/frontend/plugins/UN_template/static/css/UN_template.min.css

```diff
@@ -1 +1 @@
;font-weight:600}.gray_cell{background:#ababab;border-right:3px solid #eee;border-left:3px solid #eee}.blue_cell{border-right:3px solid #eee;border-left:3px solid #eee;border-radius:4px;background:#3bafe0}.green_cell{border-right:3px solid #eee;border-left:3px solid #eee;border-radius:4px;background:#005f72}.green_cell .horario_text{color:#fff}.orange_cell{border-right:3px solid #eee;border-left:3px solid #eee;border-radius:4px;background:#ec7708}.rl_tabs>.tab-content>.tab-pane{overflow-x:hidden!important}.content_program{background:rgba(255,255,255,.8);padding-bottom:50px;margin-bottom:20px}.row-title-program{padding:30px 0;color:#fff;background:#144242}.title-program{font-family:inherit}.content_program a,.content_program h2,.content_program h3,.content_program h4,.content_program p,.content_program ul{padding-left:10px!important;padding-right:10px!important;font-family:inherit}.subtitle_content{background:#144242;color:#fff;padding-top:15px!important;padding-bottom:15px!important;margin-top:10px!important;margin-bottom:10px!important;font-family:inherit}.font_green{color:#144242}.list_menu_lateral li a:hover{background:#fff}.list_menu_lateral li a{display:block;text-align:center;margin-bottom:5px;background:rgba(255,255,255,.8);padding:15px 10px 15px 10px;color:#144242;font-size:1.1em;text-decoration:none}.list_menu_fixed{position:fixed;z-index:1000;top:20px}.list_menu_fixed_responsive{position:fixed;z-index:1000;top:20px;left:-100%;background:#436f6f}#item-admisiones{background:rgba(20,66,66,.8)!important;color:#fff!important}#item-admisiones:hover{background:#144242!important}#info-autoevaluacion{overflow:hidden;white-space:nowrap;text-overflow:ellipsis}.candidatos div{vertical-align:middle}.candidatos{line-height:0!important}.content_program a{color:#144242!important}.content_program .accordion-heading{border:2px solid #144242!important}.content_program div[class*="accordion-group panel rl_sliders-group nn_sliders-group"]{background:0 0!important}.itemNavigation{display:none}.moduletable h3{padding:6px;text-align:center;margin-bottom:7px}.div-title-content{padding:30px 15px;color:#fff;background:#144242}.div-title-content .itemTitle{font-family:inherit;width:50%;text-align:right;text-transform:uppercase;padding:0!important}.itemBody{padding:0!important}@media screen and (min-width:300px) and (max-width:767px){.head_table .tab_text{white-space:nowrap}#arrow-right-icon{display:block!important}.div-title-content .itemTitle{width:100%}}@media screen and (min-width:768px) and (max-width:991px){.head_table .tab_text{white-space:nowrap}#arrow-right-icon{display:block!important}.div-title-content .itemTitle{width:100%}}.embed-responsive.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}
```

Comment:
Why has this file changed?

Reply:
I'm not sure. The change was that some colors passed from rgba to hex, like background: rgba (85,85,85,1) to background: #555.  It's the same color. Changed the backgrounds who had a = 1, and I don't know why. Maybe the IDE did it?